general-superstaq/dev-requirements.txt

```diff
@@ -3,6 +3,7 @@ black[jupyter]~=22.3.0
 flake8>=3.8.4
 flake8-type-checking>=2.1.0
 isort[colors]>=5.10.1
+locust>=2.15.1
```

Comment:
imo this shouldn't be part of gss's dev-requirements - we can just update the instructions at the top of locustfile.py to include `pip install locust`

Reply:
> imo this shouldn't be part of gss's dev-requirements

Reason? This is a "dev" requirement after all.

Reply:
not a big deal, but i guess i don't really think of it as a "requirement" in that it's not needed for any tests and most devs will never use it

also gss's dev-requirements are used used when installing every other superstaq package with [dev], so it adds a new dependency to five different packages for something that usually isn't really necessary for any of them

Reply:
Let's leave this until it starts causing problems in downstream packages.

Reply:
Done after IRL discussion: Locust has [a lot](https://github.com/locustio/locust/blob/17107e0fd45d84fcb0f3cfdab5a9728f45dae73a/pyproject.toml#L12) of dependencies.